specification/securitycenter/resource-manager/Microsoft.Security/preview/2017-08-01-preview/informationProtection.json

```diff
@@ -0,0 +1,360 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Information protection",
+        "description": "API spec for information protection policies",
+        "version": "2017-08-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": ["https"],
+    "consumes": ["application/json"],
+    "produces": ["application/json"],
+    "security": [{
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
```

Comment:
Is there a DELETE support for this?

Reply:
We do not support DELETE for now, as from the perspective of the user, the policy always exists (Cannot delete the built-in policy)